web/src/EducationBenchmarking.Web/Views/SchoolPlanningSteps/TimetableCycle.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@using EducationBenchmarking.Web.Extensions
+@model EducationBenchmarking.Web.ViewModels.SchoolPlanViewModel
+
+@{
+    ViewData[ViewDataConstants.Title] = PageTitleConstants.SchoolPlanningTimetableCycle;
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l">@Model.Name</span>
+        <h1 class="govuk-heading-l">@ViewData[ViewDataConstants.Title]</h1>
+        <p class="govuk-body">
+            The length of your timetable cycle is the number of periods before it repeats itself.
+        </p>
+        <p class="govuk-body">
+            For example, a timetable that repeats weekly with 5 periods per day would give you a cycle length of 25 periods
+        </p>
+        <p class="govuk-body">
+            Enter an average if your school has a different timetable length for different years
+        </p>
+        @using (Html.BeginForm("TimetableCycle", "SchoolPlanningSteps", new { urn = Model.Urn, year = Model.SelectedYear }, FormMethod.Post, true, new { novalidate = "novalidate" }))
+        {
+            <div class="@(ViewData.ModelState.HasError(nameof(Model.TimetablePeriods)) ? "govuk-form-group govuk-form-group--error" : "govuk-form-group")">
+                <div class="govuk-form-group">
+                    <label class="govuk-label govuk-label--m" for="timetable-periods">
+                        How many periods are there in one timetable cycle?
+                    </label>
+                    @if (ViewData.ModelState.HasError(nameof(Model.TimetablePeriods)))
+                    {
+                        <p id="timetable-error" class="govuk-error-message">
+                            <span class="govuk-visually-hidden">Error:</span> @ViewData.ModelState[nameof(Model.TimetablePeriods)]?.Errors.First().ErrorMessage
+                        </p>
+                    }
+                    <div class="govuk-input__wrapper">
+                        <input class="govuk-input govuk-input--width-10" id="timetable-periods" name="@nameof(Model.TimetablePeriods)" type="number" spellcheck="false" value="@((Model.TimetablePeriods != null) ? Model.TimetablePeriods : string.Empty)">
```

Comment:
Not really sure what this is doing?

`@((Model.TimetablePeriods != null) ? Model.TimetablePeriods : string.Empty)"`